django_scylla/base.py

```diff
@@ -80,7 +82,25 @@ class DatabaseWrapper(BaseDatabaseWrapper):
 
     def get_connection_params(self):
         """Return a dict of parameters suitable for get_new_connection."""
+        out_params = {}
         options = self.settings_dict.get("OPTIONS", {})
+
+        connection_options = options.get("connection", {})
+        replication_options = options.get("replication", {})
+        execution_profile_options = options.get("execution_profile", {})
+
+        if not replication_options.get("class"):
+            raise ValueError(
+                "Missing configuration value: OPTIONS.replication.class\n"
+                "See: https://docs.datastax.com/en/cassandra-oss/3.0/cassandra/architecture/archDataDistributeReplication.html"  # noqa
+            )
+
+        if not replication_options.get("replication_factor"):
+            raise ValueError(
```

Comment:
New: An explicit exception that makes configuration errors more visible to the end-user.